tests/phpunit/api/v4/Action/ContactGetTest.php

```diff
@@ -21,13 +21,15 @@
 
 use api\v4\Api4TestBase;
 use Civi\Api4\Contact;
+use Civi\Api4\ContactType;
 use Civi\Api4\Email;
 use Civi\Api4\Relationship;
 use Civi\Test\TransactionalInterface;
 
 /**
  * @group headless
  */
+
```

Comment:
@syxys1 you added a blank line here. There's no reason to do that, it's not part of your new test.

Reply:
@coleman I needed to add use Civi\Api4\ContactType; for my test.  I probably added the blank by mistake.